core/lib.rs

```diff
@@ -56,12 +55,17 @@ pub use crate::modules::RecursiveModuleLoad;
 pub use crate::normalize_path::normalize_path;
 pub use crate::ops::op_close;
 pub use crate::ops::op_resources;
+pub use crate::ops::serialize_op_result;
```

Comment:
I don't think this function should be public, what's the use case?

Reply:
I can make that private, I had left it public since it wasn't 100% clear how we'll approach plugin ops (though it seems like they won't need this and should be able to align plugins with ops without it)

Reply:
Let's make it non-public for now